src/main/java/com/openlattice/chronicle/services/delete/DataDeletionService.java

```diff
@@ -195,6 +204,21 @@ private void deleteStudyData(
         }
     }
 
+    private void ensureOwnerAccess( UUID studyId, PermissionsApi permissionsApi ) {
+        ChronicleCoreAppConfig coreAppConfig = entitySetIdsManager
+                .getLegacyChronicleAppConfig( getParticipantEntitySetName( studyId ) );
+        UUID participantsESID = coreAppConfig.getParticipantEntitySetId();
+
+        AclKey aclKey = new AclKey( participantsESID );
+
+        try {
+            permissionsApi.getAcl( aclKey );
+        } catch ( Exception e ) {
+            logger.error( "Authorization for deleting data from participant entity set {} failed", participantsESID );
+            throw new ForbiddenException( "insufficient permission to delete data from participant entity set" );
```

Comment:
`ensureOwnerAccess` does not know who the caller is so it does not know that the caller is trying to do a delete. Please update the error messages accordingly or rename the function to something like `ensureUserCanDeleteStudy`.